cppcache/src/ClientProxyMembershipID.cpp

```diff
@@ -101,12 +100,9 @@ void ClientProxyMembershipID::initObjectVars(
   m_vmViewId = vmViewId;
   m_memID.write(static_cast<int8_t>(DSCode::FixedIDByte));
   m_memID.write(static_cast<int8_t>(DSCode::InternalDistributedMember));
-  m_memID.writeArrayLen(ADDRSIZE);
-  // writing first 4 bytes of the address. This will be same until
-  // IPV6 support is added in the client
-  uint32_t temp;
-  memcpy(&temp, hostAddr, 4);
-  m_memID.writeInt(static_cast<int32_t>(temp));
+//  uint8_t * temp = new uint8_t [hostAddrLen];
```

Comment:
This pattern of allocating memory and passing it into another class to take ownership of is still found in numerous places in geode-native.  SOP is to remove it when we find it, usually by making the buffer in question a vector of the type we want, in this case uint8_t.  I took the liberty at https://github.com/pdxcodemonkey/geode-native/commit/3baf8c190430c8d25c27086223d0aced616a3aa5, feel free to cherry-pick if you like.

Reply:
Thanks